src/libs/actions/IOU.js

```diff
@@ -345,6 +347,7 @@ function getMoneyRequestInformation(
     payeeEmail = currentUserEmail,
     receipt = undefined,
     existingTransactionID = null,
```

Comment:
should be undefined

Reply:
I understand that it looks inconsistency, but `existingTransactionID` is out of scope value for this issue/PR. I am not sure that `undefined` should be, because they put `null` and it works with this default value.

Reply:
I think it should be a simple change and we can make it. And it doesn't seem to be an intentional change to use `null` Original PR which introduced the same https://github.com/Expensify/App/pull/24385/files